app/src/main/java/com/weather/weathercheckerapp/module/MainViewModelModule.kt

```diff
@@ -0,0 +1,18 @@
+package com.weather.weathercheckerapp.module
+
+import androidx.lifecycle.ViewModel
+import com.weather.weathercheckerapp.annotation.ViewModelKey
+import com.weather.weathercheckerapp.viewmodel.MainViewModel
+import dagger.Binds
+import dagger.Module
+import dagger.multibindings.IntoMap
+
+@Module
+abstract class MainViewModelModule {
```

Comment:
I think class name can be only ViewModelModule, because this class will contain more ViewModels, not only the MainViewModel.